### PR TITLE
init library

### PR DESCRIPTION
**Pull request by froosty**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,118 @@
+aiohttp_flashbag
+================
+
+The library provides flashbag for `aiohttp.web`__.
+
+.. _aiohttp_web: https://aiohttp.readthedocs.io/en/latest/web.html
```
Comment (asvetlov) [inline README.rst]: https://docs.aiohttp.org/en/stable/ :)

File: aiohttp_flashbag/__init__.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+SESSION_KEY = 'flashbag'
+
+
+def flashbag_append(request, key, value):
+    if NEW_REQUEST_KEY not in request:
+        raise RuntimeError(
+            'Install Flashbag middleware in your application',
+        )
+
+    print('=' * 100)
```
Comment (asvetlov) [inline aiohttp_flashbag/__init__.py]: `print()`?

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+import io
+import os
+import re
+
+from setuptools import setup
+
+
+def get_version():
+    with codecs.open(
+        os.path.join(
```
Comment (asvetlov) [inline setup.py]: `pathlib` looks better

**Review by asvetlov — CHANGES_REQUESTED** [review]

**Review by asvetlov — APPROVED** [review]

File: README.rst [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+
+First of all, you have to register ``aiohttp_flashbag.flashbag_middleware`` in ``aiohttp.web.Application``.
+
+You can get some data from the previous request with ``aiohttp_flashbag.flashbag_get`` method. Parameters:
+
+- **request**. Instance of ``aiohttp.web_request.Request``.
+- **key**. Name of "variable", that you want to get
+- **default**. The default value that should be returned, if the key doesn't exist in session flashbag.
+
+For specify one "variable" in flashbag you should use ``aiohttp_flashbag.flashbag_set``. Parameters:
```
Comment (roganov) [inline README.rst]: For -> To
Or better yet 'To set'.

File: README.rst [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+
+
+
+
+First of all, you have to register ``aiohttp_flashbag.flashbag_middleware`` in ``aiohttp.web.Application``.
+
+You can get some data from the previous request with ``aiohttp_flashbag.flashbag_get`` method. Parameters:
+
+- **request**. Instance of ``aiohttp.web_request.Request``.
+- **key**. Name of "variable", that you want to get
```
Comment (roganov) [inline README.rst]: Extra comma

File: README.rst [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+- **default**. The default value that should be returned, if the key doesn't exist in session flashbag.
+
+For specify one "variable" in flashbag you should use ``aiohttp_flashbag.flashbag_set``. Parameters:
+
+- **request**. Instance of ``aiohttp.web_request.Request``.
+- **key**. Name of "variable", that you want to specify.
```
Comment (roganov) [inline README.rst]: Extra comma

File: aiohttp_flashbag/__init__.py [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+    request[NEW_REQUEST_KEY] = value
+
+
+def flashbag_get(request, key, default=None):
+    if OLD_REQUEST_KEY not in request:
+        raise RuntimeError(
+            'Install Flashbag middleware in your application',
+        )
+
+    return request[OLD_REQUEST_KEY].pop(key, default)
```
Comment (roganov) [inline aiohttp_flashbag/__init__.py]: Why pop?

**Pull request merged**